.github/workflows/build.yaml

```diff
@@ -195,6 +195,9 @@ jobs:
         run: |
           ln -s vars.example vars
           mkdir ../output
+          # we need to have ability to git tag the versions
+          # we do not push that tags
+          git config --global user.name "gha_user"
```

Comment:
This is not enough. Email also needs to be configured. Still fails with `fatal: unable to auto-detect email address (got 'runner@fv-az1149-423.(none)')`
You need to merge your changes on top of my merge branch and then test the build to see whether it works. Because this only affects master builds and not release/2.6 builds.